lms/djangoapps/ccx/views.py

```diff
@@ -326,6 +327,13 @@ def override_fields(parent, data, graded, earliest=None, ccx_ids_to_delete=None)
     if changed:
         override_field_for_ccx(ccx, course, 'grading_policy', policy)
 
+    responses = SignalHandler.course_published.send(
+        sender=ccx,
```

Comment:
I understand, but I still think it is not completely right: maybe put a comment above specifying that you are using as sender the CCX object and not the CCX class: maybe the EDX reviewer will have an opinion about this.